Header header
uint32 ver
uint32 sec
uint32 usec

float64 lat
float64 lon

float64 north
float64 east
float64 z

float32 roll
float32 pitch
float32 yaw

float32 vel

uint32 nsat
uint32 ins_stat
uint32 sol
uint32 pos
string ins_str
string sol_str
string pos_str
